wezterm-input-types/src/lib.rs

```diff
@@ -1648,8 +1648,9 @@ impl KeyEvent {
         {
             // Check for simple text generating keys
             match &self.key {
+                // Don't send legacy codes with DEL and ESC
+                Char('\x1b') | Char('\x7f') => {}
                 Char('\x08') => return '\x7f'.to_string(),
```

Comment:
That's the nature of legacy terminal protocol. See the [Xterm control sequences](https://www.xfree86.org/current/ctlseqs.html#VT100%20Mode), how it sends <BS> with ^H? ^H is DEL. 

Just one of those things, like how `\n` needs to be translated by the terminal driver into `\r\n` so that the teletype in question will both return the carriage and advance the printer roll.

Reply:
Such arcane things might be good to be linked/documented in code... IMO =)